interfaces/apparmor/backend.go

```diff
@@ -839,6 +839,15 @@ func (b *Backend) addContent(securityTag string, snapInfo *snap.Info, cmdName st
 				return `#include if exists "/var/lib/snapd/apparmor/snap-tuning"`
 			}
 			return ""
+		case "###INCLUDE_IF_EXISTS_LOCAL_SNAP_PROFILE###":
+			features, _ := parserFeatures()
+			for _, f := range features {
+				if f == "include-if-exists" {
+					return fmt.Sprintf("#include if exists \"%s\"",
+						filepath.Join(dirs.SnapAppArmorDir, "local", securityTag))
+				}
+			}
```

Comment:
```suggestion
			if strutil.ListContains(parserFeatures, "include-if-exists" {
				return fmt.Sprintf("#include if exists \"%s\"",
					filepath.Join(dirs.SnapAppArmorDir, "local", securityTag))
			}
```

Reply:
Ah good catch - added in https://github.com/snapcore/snapd/pull/11619/commits/2693cf78d529cf72fe82cc57b3f097034905ab2f